src/common/utils/gpo.h

```diff
@@ -475,4 +476,9 @@ namespace powertoys_gpo {
     {
         return getUtilityEnabledValue(POLICY_ALLOW_ADVANCED_PASTE_ONLINE_AI_MODELS);
     }
+
+    inline gpo_rule_configured_t getConfiguredNewEnabledValue()
```

Comment:
The same as on the policy name. This is confusing. Please rename to `NewPlus`.

Reply:
Thank you! I've finally gotten around to renaming to NewPlus throughout. Thanks again.